f8a_worker/dispatcher/nodes.yml

```diff
@@ -477,7 +477,7 @@
       import: 'f8a_worker.storages'
       configuration:
         <<: *configurationS3
-        bucket_name: '{DEPLOYMENT_PREFIX}-manual-tags'
+        bucket_name: 'prod-manual-tags'
```

Comment:
Are you sure with this? This can introduce a lot of harm as devs from dev or staging cluster can write to production buckets.

Reply:
yes @fridex i had a discussion with @miteshvp and @samuzzal-choudhury regarding this.